runtime/test_artifacts/generator.go

```diff
@@ -161,7 +161,7 @@ func NewActors(actorType typesUtil.ActorType, n int) (actors []modules.Actor, pr
 	return
 }
 
-func GetServiceUrl(n int) string {
+func getServiceUrl(n int) string {
 	return fmt.Sprintf(ServiceUrlFormat, n)
```

Comment:
Use this function in `validatorId()`

Remove duplicate `ServiceUrlFormat` in favor of `serviceUrlFormat`

Reply:
Should I revert it to what I had before? I removed the coupling of the unit test helpers in p2p on runtime based on the previous comment:

![Screen Shot 2022-10-17 at 11 05 31 AM](https://user-images.githubusercontent.com/1892194/196250460-b2146044-6e03-42bc-a725-e8f8bb62a147.png)

Reply:
Nope - my comment is wrong. Good call out - we should not import test_artifacts here